client/src/pages/SignUp/SignUpStyles.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from "styled-components";
+
+const MainContainer = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  height: 80vh;
+  width: 30vw;
+  margin-top: 2rem;
+  padding-top: 2rem;
+  background: rgba(48, 48, 48, 0.15);
+  box-shadow: 0 8px 32px 0 rgba(48, 48, 48, 0.37);
+  backdrop-filter: blur(8.5px);
+  -webkit-backdrop-filter: blur(8.5px);
+  border-radius: 10px;
+  color: #ffffff;
+  text-transform: uppercase;
+  letter-spacing: 0.4rem;
+  @media only screen and (max-width: 320px) {
+    width: 80vw;
+    height: 90vh;
+    hr {
+      margin-bottom: 0.3rem;
+    }
+    h4 {
+      font-size: small;
+    }
+  }
+  @media only screen and (min-width: 360px) {
+    width: 80vw;
+    height: 90vh;
+    h4 {
+      font-size: small;
+    }
+  }
+  @media only screen and (min-width: 411px) {
+    width: 80vw;
+    height: 90vh;
+  }
+
+  @media only screen and (min-width: 768px) {
+    width: 80vw;
+    height: 80vh;
+  }
+  @media only screen and (min-width: 1024px) {
+    width: 70vw;
+    height: 50vh;
+  }
+  @media only screen and (min-width: 1280px) {
+    width: 30vw;
+    height: 80vh;
+  }
+`;
+
+const WelcomeText = styled.h2`
+  margin: 3rem 0 4rem 0;
+`;
+
+const InputContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 20%;
+  width: 100%;
+`;
+
+const ButtonContainer = styled.div`
+  margin: 4rem 0 2rem 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const Login = styled.h4`
+  cursor: pointer;
+`;
+
+export { MainContainer, WelcomeText, InputContainer, ButtonContainer, Login };
```

Comment:
I did it that way in case we wanted to customize each page different, making it possibly cumbersome at start but easier to change in the future, do you want me to proceed to fusion them?

Reply:
I think Login and Signup are going to be plenty similar to be able to have them fused together. Maybe not a generic input fields styles because we might want a different style on other inputs in the app, but definitely something for both login and signup.